src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -94,7 +94,7 @@
 
     // Default configs for periodic shard sync
     private static final int SHARD_SYNC_SLEEP_FOR_PERIODIC_SHARD_SYNC = 0;
-    private static final int MAX_INITIALIZATION_ATTEMPTS = 20;
+    private static final int MAX_INITIALIZATION_ATTEMPTS = 2;
```

Comment:
The fix for these long running tests was already pushed, we can rebase and undo this change

Reply:
Yeah! Sorry forgot to change this back again.